docs/index.html

```diff
@@ -44,9 +44,21 @@ <h2 class="pairport--settings-label">Difficulty Level:</h2>
     </section>
     <section id="pairport--end-game-modal" class="valign-table modal">
       <div class="valign-cell">
+	                  <div class="plane-container">
+<a href=index.html target="_blank">
```

Comment:
The value of [href] must always be inside quotes. 

Example... `<a href="https://www.google.com">Richard is awesome</a>`